internal/handlers/telegram/handler.go

```diff
@@ -120,9 +120,13 @@ photoHandler handles the Message.Photo Telegram object. Only acknowledges Photo
 exists, and sends notification to IRC
 */
 func photoHandler(tg *Client, u tgbotapi.Update) {
+	link := uploadImage(tg, u)
 	username := GetUsername(tg.IRCSettings.ShowZWSP, u.Message.From)
-	formatted := username + " shared a photo on Telegram with caption: '" +
-		u.Message.Caption + "'"
+	caption := u.Message.Caption
+	if caption == "" {
+		caption = "Untitled Message"
```

Comment:
```suggestion
		caption = "No caption provided."
```

Reply:
I feel like this is more direct English for folks on the IRC side

Reply:
makes sense